### PR TITLE
Update copyright to 2021

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ * Copyright (C) 2021 Lightbend Inc. <http://www.lightbend.com>
```
Comment (octonato) [inline journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala:2]: curious why that single file was using https.

And is that what we want? Isn't that better to just use https?

Comment (octonato) [reply journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala:2]: Checked akka/akka and we are using https. Probably also in other projects.

**Review by octonato — APPROVED** [review]
LGTM

Left some minor comments, not blocking.

File: journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala (hunk 1 above)
Comment (raboof) [reply journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala:2]: we should definitely use https - not sure what happened there :)

Comment (raboof) [reply journal/src/test/scala/akka/persistence/spanner/internal/SpannerSerializerSpec.scala]: (adding `test:headerCheck` to travis)

**Pull request merged**